### DISCUSSION TITLE
不知道是不是我电脑有啥环境没有安装吗，运行就闪退？

### DISCUSSION THREAD

**Discussion by allenlzg** [SWHL/RapidVideOCRDesktop#8] [Q&A]
请提供下述完整信息以便快速定位问题
(Please provide the following information to quickly locate the problem)
- **系统环境/System Environment**:
- **使用的是哪门语言的程序/Which programing language**:
- **所使用语言相关版本信息/Version**:
- **OnnxRuntime版本/OnnxRuntime Version**:
- **使用当前软件/库的版本/Use version**:
- **可复现问题的demo和文件/Demo of reproducible problems**:
- **完整报错/Complete Error Message**:
- **可能的解决方案/Possible solutions**:

**Comment by SWHL** [comment]
建议录个屏，逐帧看一下黑框报错什么内容，好确定哪里的问题。

**Comment by Nahuimi** [comment]
![20240422_170340](https://github.com/SWHL/RapidVideOCRDesktop/assets/46513301/68558f8a-3917-4b29-9b94-8db182e0326b)
闪退了只持续了一帧，我试了几次只有一次好像是写这traceback （most recent call last），其他都是什么都没显示就没了

**Comment by SWHL** [comment]
exe程序运行环境 兼容性较差
这个看不出来啥错误。如果会python的话，建议用命令行版的。参考教程：https://swhl.github.io/RapidVideOCR/docs/tutorial/intermediate/

**Comment by SWHL** [comment]
如果问题还存在的话， 试试v0.0.9版呗
https://github.com/SWHL/RapidVideOCRDesktop/releases/tag/v0.0.9

我手头没有Windows设备，不能测试-_-!!!!

**Comment by Nahuimi** [comment]
> 如果问题还存在的话， 试试v0.0.9版呗 https://github.com/SWHL/RapidVideOCRDesktop/releases/tag/v0.0.9
> 
> 我手头没有Windows设备，不能测试-_-!!!!

我刚试了下新版，可以了，没有闪退

**Comment by SWHL** [comment]
issue那我先关了，后续有问题再说哈

**Comment by twmht** [comment]
有沒有辦法支援 cuda

**Reply by SWHL** [reply]
有办法，但是我没试过哈，直接编译支持cuda的对应包已经就行。
